docs/qtglean.md

```diff
@@ -110,7 +110,7 @@ example:
     data_sensitivity:
       - technical
     notification_emails:
-      - brizental@mozilla.com
+      - brizental@mozilla.com, vpn-telemetry@mozilla.com
```

Comment:
I don't think this is[ valid YAML list syntax](https://yaml.org/spec/1.2.2/#block-sequences). IIUC, this should be 
```suggestion
      - brizental@mozilla.com
      - vpn-telemetry@mozilla.com
```